schema.json

```diff
@@ -29,7 +29,12 @@
     },
     "type": "object",
     "properties": {
+        "$id": {
+            "type": "string",
+            "format": "uriref"
+        },
         "id": {
+            "description": "This keyword has been deprecated in favor of \"$id\"",
```

Comment:
@awwright @epoberezkin I've submitted PR #212 as a version of this that drops `"id"` entirely.